codice/alliance
# CAL-54 Simplify the WKT Geo fields for streaming video metacard

#### What does this PR do?
The frame-center and location metacard attributes for MPEG-TS files/streams
can contain very large amounts of data. This provides a way to simplify the
data by setting a "distance tolerance" that groups near-by data points
together. The distance tolerance is configurable for both the MPEG-TS
input transformer and the MPEG-TS UDP stream monitor.

#### Who is reviewing it?
@rzwiefel @jaymcnallie @jrnorth @kcwire 
#### How should this be tested?
Build and deploy Alliance Experimental. Upload an MPEG-TS w/ KLV file mutlitple times, each time changing the distance tolerance in the mpegts input transformer metatype. Look at the raw metacard with SOLR and note how the overall number of data points change with different values for the distance tolerance. Large tolerance values (~0.01) should result in very few data points. A small tolerance (~0.0001) should result in many data points. You can also ingest the data as a UDP stream.
#### Any background context you want to provide?
#### What are the relevant tickets?
CAL-54
#### Screenshots (if appropriate)
#### Checklist:
